charts/backingservices/values.yaml

```diff
@@ -29,11 +29,22 @@ srs:
     # To use your own Elasticsearch cluster, set srsStorage.provisionInternalESCluster to false and then
     # set the external Elasticsearch cluster URL and port details below when using an externally managed elasticsearch
     # Ensure that the specified endpoint is accessible from the kubernetes cluster pods.
-    # Currently the elasticsearch connection does not support any modes of authentication and should be es endpoint APIs' accessible without authentication.
+    # Currently the elasticsearch connection is supported with Basic Authentication  and AWS IAM Role based Authentication .
 #    domain: managed-elasticsearch.acme.io
 #    port: 443
 #    protocol: https
-#    set `requireInternetAccess` to true when the elasticsearch domain is outside of the Kubernetes cluster network  and is available over internet
+#     To configure authentication for the externally managed Elasticsearch cluster to use Basic Authentication then uncomment
```

Comment:
To configure basic authentication to your externally-managed Elasticsearch cluster,  uncomment and add the parameter details: srs.srsStorage.basicAuthentication.username and srs.srsStorage.basicAuthentication.password